app/scripts/modules/core/src/search/infrastructure/infrastructure.controller.js

```diff
@@ -10,6 +10,7 @@ import { OVERRIDE_REGISTRY } from 'core/overrideRegistry/override.registry';
 import { RECENT_HISTORY_SERVICE } from 'core/history/recentHistory.service';
 import { PAGE_TITLE_SERVICE } from 'core/pageTitle/pageTitle.service';
 import { INFRASTRUCTURE_SEARCH_SERVICE } from 'core/search/infrastructure/infrastructureSearch.service';
+import { SPINNER_COMPONENT } from 'core/widgets/spinners/spinner.component';
```

Comment:
Should this be loaded at the bootstrap module level instead of at each component using it?